File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    r=[]
    for q in quiz:
        e=q.split()
        if e[1]=='+' and int(e[0])+int(e[2])==int(e[-1]):
            r+=['O']
        elif e[1]=='-' and int(e[0])-int(e[2])==int(e[-1]):
            r+=['O']
        else:
            r+=['X']  
    return r

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.cpp
#include <string>
#include <vector>
#include <sstream>
using namespace std;

vector<string> solution(vector<string> quiz) {
    vector<string> r,t;
    string buffer;
    for(int i=0;i<quiz.size();i++) {
        t={};
        stringstream ss(quiz[i]);
        while(getline(ss,buffer,' ')) {
            t.push_back(buffer);
        }
        if(t[1]=="+") {
            // if(stoi(t[0])+stoi(t[2])==stoi(t[t.size()-1]))
            //     r.push_back("O");
            // else
            //     r.push_back("X");
            r.push_back(stoi(t[0])+stoi(t[2])==stoi(t[t.size()-1])?"O":"X");
        } else {
            // if(stoi(t[0])-stoi(t[2])==stoi(t[t.size()-1]))
            //     r.push_back("O");
            // else
            //     r.push_back("X");
            r.push_back(stoi(t[0])-stoi(t[2])==stoi(t[t.size()-1])?"O":"X");
        }
    }
    return r;
}